duckduckgo/iOS
# Move Top Level Settings to SwiftUI + Other improvements

Task/Issue URL: https://app.asana.com/0/1204099484721401/1206108402009442

### Description
Getting rid of the old and ugly SettingsViewController and the related Static Storyboard View, alongside a few other changes as follows:

1. Remove SettingsViewController and a bunch of needed ViewControllers
2. Removed Storyboard views for the Top-level settings
3. Implement Top level settings using SwiftUI + MVVM approach pattern
4. Add pre-made reusable cell components for displaying standard cells, plus a "Custom" cell that accepts any view.
5. Legacy settings views (Remaining in the Settings. storyboard) are still rendered via a SwiftUI `NavigationViewController` to prevent breakage of theme and functionality
6. Moved strings to UserText and updated translations for all languages
7. Use `DesignResourcesKit` to render Settings

### How to test
- Open Settings
- Validate that all top-level switches are working
- Validate that secondary views are presented when required, and changes are reflected in the Settings View
- Check Fonts and colors for oddities.
- Validate Hidden features are hidden (Internal features not shown to users. i.e Network protection and Sync)
